src/api/output.hpp

```diff
@@ -76,6 +76,14 @@ class output_t : public wf::object_base_t
      */
     wf_pointf get_cursor_position() const;
 
+    /**
+     * Checks if a plugin can activate. This may not succeed if a plugin
+     * with the same abilities is already active or if in idle state.
```

Comment:
Not `in idle state` but if input is inhibited via input-inhibit.

Reply:
Fixed.